### PR TITLE
pkp/pkp-lib#9227 Corrent password value in user registration email

### PR DESCRIPTION
**Pull request by touhidurabir**
for pkp/pkp-lib#9227

### CODE REVIEW

File: classes/mail/mailables/UserCreated.php [hunk 1]
```diff
@@ -79,7 +82,7 @@ public function recipients(User $recipient, ?string $locale = null): Mailable
     {
         $this->traitRecipients([$recipient], $locale);
         $this->addData([
-            static::$variablePassword => $recipient->getPassword()
+            static::$variablePassword => $this->password ?? $recipient->getPassword()
```
Comment (Vitaliy-1) [inline classes/mail/mailables/UserCreated.php:85]: It looks that User::getPassword() returns encrypted password. As you passing the original password to the constructor, you can remove `$recipient->getPassword()`.

File: classes/mail/mailables/UserCreated.php [hunk 2]
```diff
@@ -49,9 +49,12 @@ class UserCreated extends Mailable
     ];
     protected static ?string $variablePassword = 'password';
 
-    public function __construct(Context $context)
+    protected ?string $password;
+
+    public function __construct(Context $context, string $password = null)
```
Comment (Vitaliy-1) [inline classes/mail/mailables/UserCreated.php:54]: Let's make the `$password` argument mandatory and add it when the mailable is instantiated, e.g., in `UserXmlPKPUserFilter::parseUser()`

**Review by Vitaliy-1 — CHANGES_REQUESTED** [review]
Thanks, @touhidurabir! Can you also fix this in `ReviewerRegister` mailable too?

File: classes/mail/mailables/UserCreated.php (hunk 1 above)
Comment (Vitaliy-1) [reply classes/mail/mailables/UserCreated.php:85]: And we need to escape it with `htmlspecialchars()`

Comment (touhidurabir) [reply classes/mail/mailables/UserCreated.php]: Done .

File: classes/mail/mailables/UserCreated.php (hunk 2 above)
Comment (touhidurabir) [reply classes/mail/mailables/UserCreated.php]: Done .

**Comment by touhidurabir** [comment]
> also fix this in ReviewerRegister mailable too?

@Vitaliy-1 done .

**Review by Vitaliy-1 — APPROVED** [review]

**Pull request merged**